packages/scene-composer/src/components/panels/scene-components/AnimationViewStateEditor.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { FormField, Button, SpaceBetween, Autosuggest } from '@awsui/components-react';
+import { useIntl } from 'react-intl';
+
+import { IAnimationComponentInternal,useEditorState, ISceneNodeInternal } from '../../../store';
+import { getAnimationObjectByName } from '../../three-fiber/AnimationComponent/AnimationComponent';
+import { useSceneComposerId } from '../../../common/sceneComposerIdContext';
+
+export type AnimationViewStateEditorProps = {
+  node: ISceneNodeInternal;
+  component: IAnimationComponentInternal;
+  onUpdate: (currentAnimations: string[], noderef: string, component: IAnimationComponentInternal) => void;
+};
+export const AnimationViewStateEditor: React.FC<AnimationViewStateEditorProps> = ({
+  node,
+  component: animationComponent,
+  onUpdate,
+}: AnimationViewStateEditorProps) => {
+  const { formatMessage } = useIntl();
+
+  const [panelnumber, setPanelNumber] = useState(0);
+  const sceneComposerId = useSceneComposerId();
+  const { getObject3DBySceneNodeRef } = useEditorState(sceneComposerId);
+  const object = getObject3DBySceneNodeRef(node.ref);
+  const animationList = object?.getObjectByName('Scene');
```

Comment:
This needs a comment explaining the magic string 'Scene' for future programmers.